CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing
+
+## Development
+
+Harpjs setup with
+
+* Live reload using browser-sync
+* JavaScript
+  * bundeling using webpack
+  * transpiling using babel
+* Linting using eslint
+
+
+```bash
+$ gulp
+$ open localhost:3000
+```
+
+
+## How this repo works
```

Comment:
This is a work in progress as stated in the title, I moved it out of the way because it needs to be rewritten and updated as it is not up to date anymore. Also details about what tech is used etc, do not belong in the main readme, just quick instructions on how to get things up and running.